frontend/src/pages/Administration.tsx

```diff
@@ -16,7 +16,7 @@ const Administration = () => {
     <Container fluid={true}>
```

Comment:
If you've removed this from the app routing just delete this file, can always get it back from source control if we want it.

Reply:
Same for the corresponding tests and snapshot.

Reply:
fixed